arduino/cores/cores.go

```diff
@@ -75,6 +75,7 @@ type PlatformRelease struct {
 	PluggableDiscoveryAware bool                          `json:"-"` // true if the Platform supports pluggable discovery (no compatibility layer required)
 	Monitors                map[string]*MonitorDependency `json:"-"`
 	MonitorsDevRecipes      map[string]string             `json:"-"`
+	Compatible              bool                          `json:"-"` // true if at all ToolDependencies are available for the current OS/ARCH.
```

Comment:
Unless necessary, I'd keep new fields private by default (we were already bitten by this practice...).
I see that you also put an `IsCompatible` method, that's 👍🏼

Reply:
Unfortunately, we cannot make it private as is used by the `packagemanager` pkg in the commit func.